common-schema.json

```diff
@@ -55,6 +55,15 @@
               }
             ]
           }
+        },
+        "seeAlso": {
```

Comment:
I think it could benefit from being either a string URI or an array of them (with minItems 1)

@fdelavega please could you update it? thanks